tests/WebexMeeting.e2e.js

```diff
@@ -47,4 +47,12 @@ describe('Meeting Widget', () => {
     MeetingPage.loadWidget(meetingDestination);
     expect(MeetingPage.meetingInfo).toBeDisplayed();
   });
+
+  it('mutes audio before joining meeting', () => {
+    MeetingPage.loadWidget(meetingDestination);
```

Comment:
We should probably add a `afterEach` where the Widget is cleaned up, if we are going to load it again on each test.
Something like `MeetingPage.unloadWidget()`